packages/offline-pay-with-crypto-widget/src/components/Modal/index.js

```diff
@@ -14,6 +38,7 @@ import cryptoIcon from 'base64-cryptocurrency-icons'
 import { ContractKitProvider } from '@celo-tools/use-contractkit'
 import '@celo-tools/use-contractkit/lib/styles.css'
 import { useContractKit } from '@celo-tools/use-contractkit'
+>>>>>>> d0b77439c9d30cb6bf7a079100ce9e81af0f72d3
```

Comment:
also here. please run the cypress tests from package.json

Reply:
how do i run cypress ?